tests/pds/dataservice_test.go

```diff
@@ -492,6 +492,132 @@ var _ = Describe("{EnableandDisableNamespace}", func() {
 
 })
 
+var _ = Describe("{ScaleDownScaleupPXCluster}", func() {
+	JustBeforeEach(func() {
+		StartTorpedoTest("ScaleDownScaleupPXCluster", "Scales Down the PX cluster, Verify Data Services and Scales up the Px cluster", pdsLabels, 0)
+	})
+
+	It("Scale down PX pods, Verify Data Services and Scale up the PX pods", func() {
+		var generateWorkloads = make(map[string]string)
+		var deployments = make(map[PDSDataService]*pds.ModelsDeployment)
+		var dsVersions = make(map[string]map[string][]string)
+		var deps []*pds.ModelsDeployment
+		var nodeList []*corev1.Node
+		torpedoNodeList := node.GetWorkerNodes()
+		Step("Deploy Data Services", func() {
+			for _, ds := range params.DataServiceToTest {
+				Step("Deploy and validate data service", func() {
+					isDeploymentsDeleted = false
+					deployment, _, dataServiceVersionBuildMap, err = DeployandValidateDataServices(ds, params.InfraToTest.Namespace, tenantID, projectID)
+					log.FailOnError(err, "Error while deploying data services")
+					deployments[ds] = deployment
+					dsVersions[ds.Name] = dataServiceVersionBuildMap
+					nodes, err := pdslib.GetNodesOfSS(*deployment.ClusterResourceName, namespace)
+					log.FailOnError(err, "Error while getting Data Serice Nodes")
+					myNode := nodes[0]
```

Comment:
rename it i.e dataServiceDeploymentNode ...

Reply:
Done